components/org.wso2.carbon.identity.api.server.application.management/org.wso2.carbon.identity.api.server.application.management.v1/src/main/resources/META-INF/cxf/applications-server-v1-cxf.xml

```diff
@@ -47,4 +52,6 @@
           class="org.wso2.carbon.identity.api.server.application.management.common.factory.STSAdminOSGiServiceFactory"/>
     <bean id="samlSsoConfigServiceFactoryBean"
           class="org.wso2.carbon.identity.api.server.application.management.common.factory.SAMLSSOConfigOSGiServiceFactory"/>
+    <bean id="templateManagerServiceFactoryBean"
+          class="org.wso2.carbon.identity.api.server.application.management.common.factory.TemplateManagementOSGiServiceFactory"/>
```

Comment:
length exceeds 120.

Reply:
In this file, the class name is given in a single line at all the other places. Therefore I used the same way.